app/task_routes.py

```diff
@@ -0,0 +1,122 @@
+from app import db
+from app.models.task import Task
+from flask import Blueprint, request, make_response, jsonify, abort
+import datetime
+import os
+import requests
+
+tasks_bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+
+# create helper function to post message to Slack
+def slack_bot(slack_message):
+    path = "https://slack.com/api/chat.postMessage"
+    SLACK_API_KEY = os.environ.get("SLACK_API_KEY")
+    HEADERS = {"Authorization": SLACK_API_KEY}
+    channel_id ="C049QSML63U"
+    
+    query_params = {
+        "channel": channel_id, 
+        "text": slack_message
+        }
+
+    requests.post(path, params=query_params, headers=HEADERS)
+
+# validate ids for models
+def validate_id(cls, model_id):
+    try: 
+        model_id = int(model_id)
+    except:
+        abort(make_response({"message":f"{cls.__name__} {model_id} invalid"}, 400))
+    
+    model = cls.query.get(model_id)
+    if not model:
+        abort(make_response({"message":f"{cls.__name__} {model_id} not found"}, 404))
+
+    return model
+
+# validate request body information
+def validate_request_body(cls, request_body):
+    try:
+        new_model = cls.from_dict(request_body)
+    except:
+        abort(make_response({"details":f"Invalid data"}, 400))
+
+    return new_model
```

Comment:
Nice! This helper could also get pulled into a separate helper file since it's used for both route files.